Extension/package.json

```diff
@@ -1102,6 +1102,12 @@
           "markdownDescription": "%c_cpp.configuration.vcpkg.enabled.markdownDescription%",
           "scope": "resource"
         },
+        "C_Cpp.addNodeAddonIncludePaths": {
+          "type": "boolean",
+          "default": false,
+          "markdownDescription": "%c_cpp.configuration.addNodeAddonIncludePaths.description%",
+          "scope": "resource"
```

Comment:
On second thought, I think this should be "application" scope (i.e. User scope in the UI), because otherwise it would be impossible for a user to open/browse a workspace from the internet securely, because execChildProcess is called. Also, it seems rare to want to be enable/disable the setting on a per-project basis anyway.

Reply:
ok. the scope definitions are still a bit of a mystery to me but next push will have this change.